pages/how-to-apply.md

```diff
@@ -2,12 +2,10 @@
 permalink: /how-to-apply/
 title: How to apply
 ---
+If you are interested in joining 18F, use the application link below. This page also contains other information about the application process.
```

Comment:
@fureigh I don't think we need it. I put it in there because it felt a little empty after taking out the content. But I'm good with just having the application link,